Realm/Realm/Configurations/SyncConfiguration.cs

```diff
@@ -123,6 +123,7 @@ private SyncConfiguration(object partition, User user, string path)
         internal override Realm CreateRealm(RealmSchema schema)
         {
             var configuration = CreateConfiguration();
+            configuration.additive_discovered = ObjectClasses == null ? true : false;
```

Comment:
You don't need the ternary here. `ObjectClasses == null` is already a boolean.

Reply:
🤦🏻